compiler/plc_ast/src/ast.rs

```diff
@@ -614,7 +615,7 @@ pub enum AstStatement {
     CaseCondition(Box<AstNode>),
     ExitStatement(()),
     ContinueStatement(()),
-    ReturnStatement(()),
+    ReturnStatement(ReturnStatement),
```

Comment:
I would also add this to the debug impl : 
if condition is not None we forward to it under "Condition" othewise we keep the current behaviour
We can do that in a different commit no need to keep this blocked however.